File: src/main/java/com/cfgglobal/test/domain/SysConfig.java
package com.cfgglobal.test.domain;

import lombok.AccessLevel;
import lombok.Data;
import lombok.experimental.FieldDefaults;
import org.hibernate.annotations.DynamicInsert;
import org.hibernate.annotations.DynamicUpdate;

import javax.persistence.Entity;

@Entity
@Data
@DynamicUpdate
@DynamicInsert
@FieldDefaults(level = AccessLevel.PRIVATE)
public class SysConfig extends BaseEntity {
    String confKey;
    String confVal;
}

File: src/main/kotlin/com/cfgglobal/test/domain/SysConfig.kt
package com.cfgglobal.test.domain

import org.hibernate.annotations.DynamicInsert
import org.hibernate.annotations.DynamicUpdate
import javax.persistence.Entity

@Entity
@DynamicUpdate
@DynamicInsert
data class SysConfig(
        var confKey: String = "",
        var confVal: String = ""
) : BaseEntity()
